### PR TITLE
Aman

### PR DESCRIPTION
**Pull request by Amanopia**

### CODE REVIEW

**Comment by Amanopia** [comment]
https://github.com/opencog/TinyCog/pull/33/commits/8852e7b1956916d18102beef28f46bf7aa20e7ad

File: act/manual/manualCommands.cpp [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+/*
+
+* Assumption: position is between 0  and 45 degrees. negetive degrees imply rotation in reverse direction.
+            : speed is between 
+* SPEEDVOLTAGECONSTANT is minimum delay of the motor that could bring the maximum rotation(position) multiplied by 100.
+ It is dependent on voltage supplied. forexample 1800 is SPEEDVOLTAGECONSTANT for 5V supply
+* 'isOpened' boolean return true for objects opened more than 11 degrees else it returns false 
+* currentPosition is valid only the whole life of one uploaded program, else it returns zero.
+* It is better always to start by resetAll(), unless CurrentPosition &  Absolute move commands may not bring expected results.
+* same pwm pin is connected for all bodys i.e it is not possible to differnt body motors wit differnt speeds at the same time.
```
Comment (linas) [inline act/manual/manualCommands.cpp]: Lines should be less than 80 characters long -- 72 characters is typical.

File: act/manual/manualCommands.cpp [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+{eye,{12,13}},
+{forehead,{10,11}},
+{check,{15,16}},
+};
+map<std::string,int> feedbackPinStore ={
+{mouth,19},
+{head,21},
+{forehead,22},
+{eye,23},
+{check,24}
```
Comment (linas) [inline act/manual/manualCommands.cpp]: Using standard indentation would be nice ...

**Comment by linas** [comment]
Pull request titles that describe what is actually in the pull request is best!

I mean, this seems like reasonable work; I am not carefully following this project ... and I know you closed this pull req instead of merging... but still, "normal" conventions should be followed.

**Comment by Amanopia** [comment]
thanks!

**Pull request closed without merging**